### PR TITLE
[MIG] crm_lead_website: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Migrate tweaks
- Adapt methodsº

cc @Tecnativa

### CODE REVIEW

File: crm_lead_website/models/crm_lead.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
```
Comment (yajo) [inline crm_lead_website/models/crm_lead.py]: No longer Antiun's.

**Review by yajo — APPROVED** [review]
Little comment.

Would be nice if you squash translation commits.

File: crm_lead_website/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+This module was written to extend the functionality of CRM leads to support
+setting the website.
+
+Configuration
+=============
+
+To see some effects of this module, you need to enable leads:
+
+#. Enable *Sales > Configuration > Settings > Use leads if you need a
+   qualification step before creating an opportunity or a customer*.
```
Comment (luismontalba) [inline crm_lead_website/README.rst]: CRM > Configuration > Settings

**Review by luismontalba — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

**Pull request merged**